public/app/plugins/datasource/loki/components/LokiCheatSheet.tsx

```diff
@@ -37,7 +39,10 @@ const LOGQL_EXAMPLES = [
   },
 ];
```

Comment:
Thanks a lot for this clean up! I wasn't aware we still had these legacy styles.